docs/curation-review-tut.md

```diff
@@ -0,0 +1,570 @@
+
+- [`seurat2anndata()`](https://github.com/cellgeni/sceasy/blob/f8f0628a280e0880ea94b00100b463e1f6ba1994/R/functions.R#L14)
+- [`sce2anndata()`](https://github.com/cellgeni/sceasy/blob/f8f0628a280e0880ea94b00100b463e1f6ba1994/R/functions.R#L64)
+- [`loom2anndata()`](https://github.com/cellgeni/sceasy/blob/f8f0628a280e0880ea94b00100b463e1f6ba1994/R/functions.R#L116)
+
+
+**Note:** While `AnnData` is able to accomodate multiple representations of an expression matrix in the object, matrices that are stored in `adata.layers` are required to be the same dimensions as `adata.X` (`adata.raw.X` may be of a different dimensionality though). In some scenarios, you may need to construct different `AnnData` objects to accomodate different `assays` in the same experiment (for example, spliced vs unspliced counts in a sNuc-seq experiment).
+
+<br/>
+
+### Alternative assays
+
+The cellxgene data portal and explorer are able to handle a wide range of single cell data types. Due to requirements of the cellxgene schema and limitations of the `AnnData` object structure, there are some assay specific considerations that are outlined below
+
+- ATAC/mC
+  - raw data: Since there is not a standard way to generate a counts matrix or gene activity matrix for single cell epigenetic data, it is suitable to put a copy of `adata.X` in place of a raw data matrix. You can specify this assignment in `adata.uns`
+  - [(Not Live)Link to example curated dataset](www.example.com)
+- RNAseq
+  -  In some single cell toolchains, outputs of different computational methods may be of a different dimensionality than the input matrix (i.e SCTransform, or the scale.data slot in Seurat objects). Since `AnnData` objects do not allow for matrices of different dimensions in `adata.layers`, it is suitable to pad the missing rows/features with zeros to ensure that indices (feature names) across different the different layers are matched. You can specify that these matrices were padded in `adata.uns['layer_descriptions']` as a part of the cellxgene schema. This allows users who may reuse your data to remove the padding from the matrix before working with the data further.
+  -  [(Not Live)Link to example curated dataset](www.example.com)
+- CITEseq
+  - TODO: considerations for CITE-seq...
+  - [(Not Live)Link to example curated dataset](www.example.com)
+- Spatial/Visium
+  - Since spatial sequencing spots are not at the single cell level, you may wish to include prediction matrices that provide deconvolution information for each spot. These can generally be stored in `adata.uns` although it can be interesting to add prediction scores for each spot as new columns to the `adata.obs` dataframe
```

Comment:
_it can be interesting_ ... because ?

Reply:
updated with:

...although it can be interesting to add prediction scores for each spot as new columns to the `adata.obs` dataframe (to get a cell type score for each spot)...